integration/e2e/db/db.go

```diff
@@ -109,3 +109,14 @@ func NewCassandra() *e2e.HTTPService {
 		9042,
 	)
 }
+
+func NewSwiftStorage() *e2e.HTTPService {
+	return e2e.NewHTTPService(
+		"swift",
+		images.SwiftEmulator,
+		nil,
+		// readiness probe inspired from https://github.com/kubernetes/examples/blob/b86c9d50be45eaf5ce74dee7159ce38b0e149d38/cassandra/image/files/ready-probe.sh
+		e2e.NewHTTPReadinessProbe(8080, "/", 404, 404),
```

Comment:
Do we have any better way to test for readiness? We also support command probes.

Reply:
I feel like this is a good one for the use case, waiting for the HTTP api to be responsive. That's all we need for the e2e.